packages/landing/src/components/debug.tsx

```diff
@@ -383,10 +390,13 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
     const isTurnOff = sourceId === 'off' || sourceId == null;
     const currentTerrain = map.getTerrain();
     if (isTurnOff) {
+      Config.map.setDebug('debug.terrain', null);
```

Comment:
should L387 be removed if this is done here

Reply:
I think we should keep this, it doesn't make sense to` debug.terrain=off` when selecting `off` in dropdown, we just set it as `null` should same meaning.
Also, if we don't set null here, line 124 also need to check `null` and `off`.